src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,19 @@
-export const Tabs = () => {};
+export const Tabs = ({ tab, selectedTabId, onTabSelected, setTabItem }) => (
+  <li
+    data-cy="Tab"
+    className={`${selectedTabId === tab.id && 'is-active'}`}
```

Comment:
The className value is a boolean type, which could lead to unexpected results. You should convert it into a string using a ternary operator or another method.